mne/time_frequency/csd.py

```diff
@@ -255,3 +221,187 @@ def compute_epochs_csd(epochs, mode='multitaper', fmin=0, fmax=np.inf,
                                              frequencies=frequencies[i],
                                              n_fft=n_fft))
         return csds
+
+
+@verbose
+def csd_array(X, sfreq, mode='multitaper', fmin=0, fmax=np.inf,
+              fsum=True, n_fft=None, mt_bandwidth=None,
+              mt_adaptive=False, mt_low_bias=True, verbose=None):
+    """Estimate cross-spectral density from an array.
+
+    .. note:: Results are scaled by sampling frequency for compatibility with
+              Matlab.
+
+    Parameters
+    ----------
+    X : array-like, shape (n_replicates, n_series, n_times)
+        The time series data.
+    sfreq : float
+        Sampling frequency of observations.
+    mode : str
+        Spectrum estimation mode can be either: 'multitaper' or 'fourier'.
```

Comment:
@Eric89GXL @kingjr There does not seem to be a consistent style in mne-python about whether default values are in the docstring or are only in the function definition. I am a fan of not duplicating them in the docstring as the function definition is basically always printed with the documentation and it's just another thing to forget to change if the default value is ever changed.